src/emojis.ts

```diff
@@ -0,0 +1,1884 @@
+  man_in_steamy_room: "🧖‍♂️",
+  person_in_steamy_room: "🧖‍♂️",
+  woman_climbing: "🧗‍♀️",
+  person_climbing: "🧗‍♀️",
+  man_climbing: "🧗‍♂️",
+  woman_in_lotus_position: "🧘‍♀️",
+  person_in_lotus_position: "🧘‍♀️",
+  man_in_lotus_position: "🧘‍♂️",
+  female_mage: "🧙‍♀️",
+  mage: "🧙‍♀️",
+  male_mage: "🧙‍♂️",
+  female_fairy: "🧚‍♀️",
+  fairy: "🧚‍♀️",
+  male_fairy: "🧚‍♂️",
+  female_vampire: "🧛‍♀️",
+  vampire: "🧛‍♀️",
+  male_vampire: "🧛‍♂️",
+  mermaid: "🧜‍♀️",
+  merman: "🧜‍♂️",
+  merperson: "🧜‍♂️",
+  female_elf: "🧝‍♀️",
+  male_elf: "🧝‍♂️",
+  elf: "🧝‍♂️",
+  female_genie: "🧞‍♀️",
+  male_genie: "🧞‍♂️",
+  genie: "🧞‍♂️",
+  female_zombie: "🧟‍♀️",
+  male_zombie: "🧟‍♂️",
+  zombie: "🧟‍♂️",
+  brain: "🧠",
+  orange_heart: "🧡",
+  billed_cap: "🧢",
+  scarf: "🧣",
+  gloves: "🧤",
+  coat: "🧥",
+  socks: "🧦",
+  red_envelope: "🧧",
+  firecracker: "🧨",
+  jigsaw: "🧩",
+  test_tube: "🧪",
+  petri_dish: "🧫",
+  dna: "🧬",
+  compass: "🧭",
+  abacus: "🧮",
+  fire_extinguisher: "🧯",
+  toolbox: "🧰",
+  bricks: "🧱",
+  magnet: "🧲",
+  luggage: "🧳",
+  lotion_bottle: "🧴",
+  thread: "🧵",
+  yarn: "🧶",
+  safety_pin: "🧷",
+  teddy_bear: "🧸",
+  broom: "🧹",
+  basket: "🧺",
+  roll_of_paper: "🧻",
+  soap: "🧼",
+  sponge: "🧽",
+  receipt: "🧾",
+  nazar_amulet: "🧿",
+  ballet_shoes: "🩰",
+  "one-piece_swimsuit": "🩱",
+  briefs: "🩲",
+  shorts: "🩳",
+  thong_sandal: "🩴",
+  drop_of_blood: "🩸",
+  adhesive_bandage: "🩹",
+  stethoscope: "🩺",
+  "yo-yo": "🪀",
+  kite: "🪁",
+  parachute: "🪂",
+  boomerang: "🪃",
+  magic_wand: "🪄",
+  pinata: "🪅",
+  nesting_dolls: "🪆",
+  ringed_planet: "🪐",
+  chair: "🪑",
+  razor: "🪒",
+  axe: "🪓",
+  diya_lamp: "🪔",
+  banjo: "🪕",
+  military_helmet: "🪖",
+  accordion: "🪗",
+  long_drum: "🪘",
+  coin: "🪙",
+  carpentry_saw: "🪚",
+  screwdriver: "🪛",
+  ladder: "🪜",
+  hook: "🪝",
+  mirror: "🪞",
+  window: "🪟",
+  plunger: "🪠",
+  sewing_needle: "🪡",
+  knot: "🪢",
+  bucket: "🪣",
+  mouse_trap: "🪤",
+  toothbrush: "🪥",
+  headstone: "🪦",
+  placard: "🪧",
+  rock: "🪨",
+  fly: "🪰",
+  worm: "🪱",
+  beetle: "🪲",
+  cockroach: "🪳",
+  potted_plant: "🪴",
+  wood: "🪵",
+  feather: "🪶",
+  anatomical_heart: "🫀",
+  lungs: "🫁",
+  people_hugging: "🫂",
+  blueberries: "🫐",
+  bell_pepper: "🫑",
+  olive: "🫒",
+  flatbread: "🫓",
+  tamale: "🫔",
+  fondue: "🫕",
+  teapot: "🫖",
+  bangbang: "‼️",
+  interrobang: "⁉️",
+  tm: "™️",
+  information_source: "ℹ️",
+  left_right_arrow: "↔️",
+  arrow_up_down: "↕️",
+  arrow_upper_left: "↖️",
+  arrow_upper_right: "↗️",
+  arrow_lower_right: "↘️",
+  arrow_lower_left: "↙️",
+  leftwards_arrow_with_hook: "↩️",
+  arrow_right_hook: "↪️",
+  watch: "⌚",
+  hourglass: "⌛",
+  keyboard: "⌨️",
+  eject: "⏏️",
+  fast_forward: "⏩",
+  rewind: "⏪",
+  arrow_double_up: "⏫",
+  arrow_double_down: "⏬",
+  black_right_pointing_double_triangle_with_vertical_bar: "⏭️",
+  black_left_pointing_double_triangle_with_vertical_bar: "⏮️",
+  black_right_pointing_triangle_with_double_vertical_bar: "⏯️",
```

Comment:
I mean, at this point it's maybe easier to use the actual emoji :joy:

Reply:
Done in [fec0afa](https://github.com/radicle-dev/radicle-interface/pull/357/commits/fec0afae55a9df11e7e5d1f6fb1f38fa4dbe09be)